packages/styled-docs/pages/flex.mdx

```diff
@@ -1 +1,205 @@
 # Flex
+
+Flex is [Box](box) with `display: flex` and comes with helpful style shorthand props.
+
+## Import
+
+```js
+import Flex from '@trcolumn-reversemicro/react-styled-core/Flex';
```

Comment:
Is the "@trcolumn-reversemicro"  should be "@trendmicro"?

Reply:
DevOooooooooops.....